docs/troubleshoot/zowe-chat-troubleshoot/troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+# Troubleshooting Zowe Chat
+
+As a Zowe Chat user, you might encounter some issues with how Zowe Chat works. This section lists some solutions to help you. 
+
+## Check the chatServer.log
+
+When you run into some errors, you can check the server log which can be found in `$ZOWE_CHAT_HOME/chatServer.log/`. 
```

Comment:
Hi. log directory is $ZOWE_CHAT_HOME/log and log file is $ZOWE_CHAT_HOME/log/chatServer.log